cmd/askd/handlers/form_submission.go

```diff
@@ -253,3 +307,25 @@ func (formSubmissionHandle) RemoveFlag(c *app.Context) error {
 
 	return nil
 }
+
+// encodeCSV gets all the submissions and encode them into a CSV
+func encodeCSV(s []submission.Submission) ([]byte, error) {
+
+	var buf bytes.Buffer
+	w := csv.NewWriter(&buf)
+	if err := w.Write(s[0].GetQuestions()); err != nil {
```

Comment:
What I would expect is that for each submission, you'd determine the question that was asked (perhaps based on the id) and ensure that there is enough columns for each of the questions, and write each submission line by line with the correct amount of blank spaces for questions not provided by the submission.